ocpp/v201/enums.py

```diff
@@ -1350,3 +1351,1347 @@ class SecurityEventType(str, Enum):
+    """
+
+    availability_state = "AvailabilityState"
+    available = "Available"
+    charge_protocol = "ChargeProtocol"
+    connector_type = "ConnectorType"
+    enabled = "Enabled"
+    phase_rotation = "PhaseRotation"
+    problem = "Problem"
+    supply_phases = "SupplyPhases"
+    supply_phases_max_limit = "SupplyPhases(MaxLimit)"
+    tripped = "Tripped"
+
+
+class ConnectorHolsterReleaseVariableName(str, Enum):
+    """
+    Variable names where the component type is ConnectorHolsterRelease
+    See PhysicalComponentName for referenced physical component
+    """
+
+    enabled = "Enabled"
+    active = "Active"
+    problem = "Problem"
+    state = "State"
+
+
+class ConnectorHolsterSensorVariableName(str, Enum):
+    """
+    Variable names where the component type is ConnectorHolsterSensor
+    See PhysicalComponentName for referenced physical component
+    """
+
+    enabled = "Enabled"
+    active = "Active"
+    problem = "Problem"
+
+
+class ConnectorPlugRetentionLockVariableName(str, Enum):
+    """
+    Variable names where the component type is ConnectorPlugRetentionLock
+    See PhysicalComponentName for referenced physical component
+    """
+
+    enabled = "Enabled"
+    active = "Active"
+    problem = "Problem"
+    tripped = "Tripped"
+    tries = "Tries"
+    tries_set_limit = "Tries(SetLimit)"
+    tries_max_limit = "Tries(MaxLimit)"
+
+
+class ConnectorProtectionReleaseVariableName(str, Enum):
+    """
+    Variable names where the component type is ConnectorProtectionRelease
+    See PhysicalComponentName for referenced physical component
+    """
+
+    enabled = "Enabled"
+    active = "Active"
+    problem = "Problem"
+    tripped = "Tripped"
+
+
+class ControllerVariableName(str, Enum):
+    """
+    Variable names where the component type is Controller
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    ec_variant = "ECVariant"
+    firmware_version = "FirmwareVersion"
+    interval_heartbeat = "Interval[Heartbeat]"
+    manufacturer = "Manufacturer"
+    max_msg_elements = "MaxMsgElements"
+    model = "Model"
+    problem = "Problem"
+    selftest_active = "SelftestActive"
+    selftest_active_set = "SelftestActive(Set)"
+    serial_number = "SerialNumber"
+    version_date = "VersionDate"
+    version_number = "VersionNumber"
+
+
+class ControlMeteringVariableName(str, Enum):
+    """
+    Variable names where the component type is ControlMetering
+    See PhysicalComponentName for referenced physical component
+    """
+
+    power = "Power"
+    ac_current = "ACCurrent"
+    dc_current = "DCCurrent"
+    dc_voltage = "DCVoltage"
+
+
+class CPPWMControllerVariableName(str, Enum):
+    """
+    Variable names where the component type is CPPWMController
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    dc_voltage = "DCVoltage"
+    enabled = "Enabled"
+    percent = "Percent"
```

Comment:
In OCPP-2.0.1_part2_appendices_v13.pdf, 3.2.21. CPPWMController that's `Percentage`. I can't find a similar entry in the CSVs.

Reply:
In this case, I thought it was a mistake - as the generic type is percent - it is only this one which calls it percentage ( in the description ) - I'll change it and if it is fixed in an Errata then it can be updated